srcpkgs/qsynth/template

```diff
@@ -1,20 +1,14 @@
 # Template file for 'qsynth'
 pkgname=qsynth
-version=0.9.2
-revision=2
-build_style=gnu-configure
-build_helper=qmake
-hostmakedepends="pkg-config qt5-tools"
-makedepends="qt5-x11extras-devel jack-devel fluidsynth-devel"
-depends="fluidsynth jack desktop-file-utils hicolor-icon-theme"
+version=0.9.13
+revision=1
+build_style=cmake
+hostmakedepends="pkg-config qt6-tools qt6-base"
+makedepends="qt6-base-devel qt6-svg-devel fluidsynth-devel"
+depends="fluidsynth"
```

Comment:
```suggestion
depends="fluidsynth jack"
```

This still depends on JACK at runtime when I tested it.

Reply:
ye, but jack server is also provided by pipewire

Reply:
Yeah, fair enough.